Packs/RubrikPolaris/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,70 @@
+- Various layout changes were made to the Rubrik Radar Incident Summary page.
+- Renamed the Rubrik Polaris CDM Cluster ID field to Rubrik CDM Cluster ID and modified the associated mapper.
+- Added the RubrikRadarFilesAdded, RubrikRadarFilesModified and RubrikRadarFilesDeleted scripts which pulls in the respective Radar File Change attributes.
+- Added the RubrikCDMClusterConnectionState script that pulls in the connection status for the CDM cluster.
+- Added the RubrikSonarTotalHits script to pull in the total number of Sonar Data Classification Hits.
+- Added the RubrikSonarOpenAccessFiles script to pull the open access permissions file count from Sonar.
+- Added the Rubrik Post Intrusion Ransomware Investigation playbook.  This has been made the default playbook for the Rubrik Radar incident type.
+- Added the Rubrik Polaris tab to the Rubrik Radar incident layout.  This new tab includes the Anomaly Detection Event Timeline and Data Classification sections.
+- Modified the Radar Mapper to now map the Hostnames field to the Radar Polaris "objectName" field of the Radar event.
+- Removed the Rubrik Analysis Progress, Rubrik Polaris Event Completed, and Rubrik Polaris Last Updated custom fields.
+- Removed the rubrik_radar_analysis_status command and playbook due to them being no longer needed.
+- Updated the Docker image to: *demisto/python3:3.9.4.18682*.
+
+#### Scripts
+##### New: RubrikCDMClusterConnectionState
+- Shows the Rubrik Radar amount of Files Added.
+
+##### New: RubrikRadarFilesAdded
+- Shows the Rubrik Radar amount of Files Added.
+
+##### New: RubrikRadarFilesDeleted
+- Shows the Rubrik Radar amount of Files Deleted.
+
+##### New: RubrikRadarFilesModified
+- Shows the Rubrik Radar amount of Files Modified.
+
+##### New: RubrikSonarOpenAccessFiles
+- Shows the Rubrik Polaris Sonar Open Access Files Count.
+
+##### RubrikSonarSensitiveHits
+- Shows the data classification hits on an object.
+
+##### New: RubrikSonarTotalHits
+- Shows the Rubrik Polaris Sonar Total Hits.
+
+#### Playbooks
+##### Rubrik Polaris - Anomaly Analysis
+ - Added the rubrik-cdm-cluster-location and the rubrik-cdm-cluster-connection-state tasks into the playbook under the Sonar Data Classification section.
+
+#### Mappers
+##### Rubrik Polaris Radar - Mapping
+- Varius changes to the mapping file to accomodate new fields.
+
+#### Layouts
+##### Rubrik Polaris Radar
+- Added the Rubrik Polaris tab that contains the Radar event timeline as well as the Data Classification sections.
+- Various layout changes to the Incident Summary tab.
+
+#### Incident Fields
+- **Rubrik CDM Cluster ID**
+- **Rubrik CDM Cluster Location**
+- **Rubrik CDM Cluster Name**
+- **Rubrik Polaris Analysis Pipeline**
+- **Rubrik Polaris Analysis Progress**
+- **Rubrik Polaris CDM Cluster ID**
+- **Rubrik Polaris CDM Cluster Name**
+- **Rubrik Polaris Last Updated**
+- **Rubrik Polaris Message**
+
+#### Incident Types
+##### Rubrik Radar
+- The default integration type was renamed.
```

Comment:
```suggestion
- Updated the default integration type.
```

Reply:
Thank you @roysagi!  Committed as many of these suggested changes directly here.  For some reason it's not letting me commit the last one directly, so pushed up this commit.